src/components/Calculator/CalculatorModel.ts

```diff
@@ -0,0 +1,84 @@
+import { Digit, OPERATION, MAX_LENGTH } from 'constants/calculator';
+import type { ValueOf } from 'types';
+
+type Operation = ValueOf<typeof OPERATION>;
+
+const calculationOptions = {
+  [OPERATION.PLUS]: (a: number, b: number) => a + b,
+  [OPERATION.MINUS]: (a: number, b: number) => a - b,
+  [OPERATION.MULTIPLE]: (a: number, b: number) => a * b,
+  [OPERATION.DIVIDE]: (a: number, b: number) => Math.floor(a / b),
+};
+
+interface ICalculator {
+  operation: Operation | null;
+  leftOperand: number;
+  rightOperand: number | null;
+  accumulator: string;
+}
```

Comment:
```suggestion
interface ICalculator {
  operation?: Operation;
  leftOperand: number;
  rightOperand?: number;
  accumulator: string;
}
```
이렇게 옵셔널로 표현해줄 수도 있을 것 같아요!
다만 이럴 때는 null이 아니라 undefined로 관리해야겠네요..!